Source/Windows/Common/PortabilityInfo.h

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+#pragma once
+#include "Common/Config.h"
+
+namespace FEX {
+static inline FEX::Config::PortableInformation ReadPortabilityInformation(const fextl::string& ExecutablePath) {
```

Comment:
Drop arg and the GetExecutableFileName method, also comment below is outdated

Reply:
Right, fixed.